app/components/hotkey-map/align.element.js

```diff
@@ -68,6 +77,7 @@ export class AlignHotkeys extends HotkeyMap {
       <span side>${side}</span>
```

Comment:
what do you think of this string as the display code for the keyboard interactions?

```
      <span tool>${this._tool}</span>
      <span light> as </span>
      <span>${negative_modifier}:</span>
      <span side>${side}</span>
      <span light>, distributed </span>
      <span amount>${amount},</span>
      <span light> with </span>
      <span>${negative}</span>
```

Reply:
takes more space but it also makes clearer what's going on, I like it

Reply:
i agree it's starting to take up a lot of space, there's some UI work for me to do too so the keyboard doesnt stretch to the width of the text above it. minor stuff though really. 

i think this PR is just pending a commit that updates the string with the above suggestions, and then it'd good to go.

Reply:
i committed updates